tests/mir-opt/dest-prop/dead_stores_better.rs

```diff
@@ -12,6 +11,13 @@ fn id<T>(x: T) -> T {
 
 // EMIT_MIR dead_stores_better.f.DestinationPropagation.after.mir
 pub fn f(mut a: usize) -> usize {
+    // CHECK-LABEL: fn f(
+    // CHECK: debug a => [[a:_.*]];
+    // CHECK: debug b => [[b:_.*]];
+    // CHECK: [[b]] = [[a]];
+    // CHECK: [[a]] = const 5_usize;
```

Comment:
Shouldn't DSE have removed this?

Reply:
I was surprised too, but the test was like this